### DISCUSSION TITLE
Firefox GX + Arkenfox

### DISCUSSION THREAD

**Discussion by Rev-0** [Godiesc/firefox-gx#76] [Q&A]
Hey, I want to use Firefox gx but I have to copy a user.js file but I already have another user.js file (Arkenfox).
Can I use multiple user.js or how do I combine both ?

**Comment by Godiesc** [comment]
hi,
You can copy the code inside my user.js and paste at the end of your other user.js and that will work, or you can manually change in about:config the rules of here: https://github.com/Godiesc/firefox-gx/blob/main/user.js
